snippets/kickstart_networking_setup.erb

```diff
@@ -113,8 +116,14 @@ DEVICE="$real"
 HWADDR="<%= interface.mac -%>"
 <% end -%>
 ONBOOT=yes
+<% if interface.respond_to?(:primary) && interface.primary -%>
+DEFROUTE=yes
```

Comment:
this is a PPP-specific option, I don't think it's relevant as the route'd be determined by the gateway?